app/models/responsible_body.rb

```diff
@@ -177,27 +180,11 @@ def is_ordering_for_schools?
   end
 
   def is_ordering_for_all_schools?
-    active_schools.count == active_schools.that_are_centrally_managed.count
-  end
-
-  def laptops_available_to_order
-    std_device_pool&.devices_available_to_order.to_i
+    active_schools.count == active_schools.responsible_body_will_order_devices.count
```

Comment:
Prefer `size` over count as `count` will always trigger a db query.

Reply:
Changed to .size after our conversation